test/cri-containerd/container_test.go

```diff
@@ -893,3 +893,56 @@ func Test_CreateContainer_HugePageMount_LCOW(t *testing.T) {
 		t.Fatalf("output is supposed to contain pagesize=2M, output: %s", output)
 	}
 }
+
+func Test_RunContainer_ExecUser_LCOW(t *testing.T) {
```

Comment:
Could you add another test which tries to run the exec with root user? Basically, any user other than the one defined in the image.

Reply:
Sure

Reply:
Added!